doc/user/content/sql/create-table.md

```diff
@@ -61,50 +68,99 @@ Tables do not currently support:
 - Unique constraints
 - Check constraints
 
-See also the known limitations for [`INSERT`](../insert#known-limitations),
-[`UPDATE`](../update#known-limitations), and [`DELETE`](../delete#known-limitations).
+See also the known limitations for [`INSERT`](/sql/insert#known-limitations),
+[`UPDATE`](/sql/update#known-limitations), and [`DELETE`](/sql/delete#known-limitations).
 
-### Temporary tables
+## PostgreSQL source tables
 
-The `TEMP`/`TEMPORARY` keyword creates a temporary table. Temporary tables are
-automatically dropped at the end of the SQL session and are not visible to other
-connections. They are always created in the special `mz_temp` schema.
+### Table names and column names
 
-Temporary tables may depend upon other temporary database objects, but non-temporary
-tables may not depend on temporary objects.
+Names for tables and column(s) must follow the [naming
+guidelines](/sql/identifiers/#naming-restrictions).
 
-## Examples
+<a name="supported-db-source-types"></a>
+
+### Read-only tables
+
+{{< include-md file="shared-content/create-table-from-source-readonly.md" >}}
 
-### Creating a table
+### Source-populated tables and snapshotting
 
-You can create a table `t` with the following statement:
+{{< include-md file="shared-content/create-table-from-source-snapshotting.md"
+>}}
 
-```mzsql
-CREATE TABLE t (a int, b text NOT NULL);
-```
+### Supported data types
 
-Once a table is created, you can inspect the table with various `SHOW` commands.
+{{< include-md file="shared-content/postgres-supported-types.md" >}}
 
-```mzsql
-SHOW TABLES;
-TABLES
-------
-t
+{{< include-md file="shared-content/postgres-unsupported-types.md" >}}
 
-SHOW COLUMNS IN t;
-name       nullable  type
--------------------------
-a          true      int4
-b          false     text
-```
+### Handling table schema changes
+
+The use of [`CREATE SOURCE`](/sql/create-source/postgres-v2/) with `CREATE TABLE
+FROM SOURCE` allows for the handling of upstream DDL changes without downtime.
+
+### Upstream table truncation restrictions
+
+{{< include-md file="shared-content/postgres-truncation-restriction.md" >}}
```

Comment:
single-sourced.  Does this still apply that you can't truncate tables upstream or it'll bork the whole source?